### PR TITLE
Update Module Reference

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request (PR) replace 2.4.6 topics with 2.4.7 topics in the Module Reference. Also, cleans up 2.4.7-beta remains.

## Affected pages

- https://developer.adobe.com/commerce/php/module-reference/

### CODE REVIEW

**Comment by dshevtsov** [comment]
@keharper can you please help me with a dead link https://developer.adobe.com/commerce/webapi/graphql/usage/is-email-available.html? It's in `module-customer-graph-ql` in Module Reference.

**Comment by dshevtsov** [comment]
> @keharper can you please help me with a [dead link](https://github.com/AdobeDocs/commerce-php/actions/runs/8602178073/job/23571128075#step:9:15) https://developer.adobe.com/commerce/webapi/graphql/usage/is-email-available.html? It's in `module-customer-graph-ql` in Module Reference.

I found https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/is-email-available. Is that correct link? Was this topic replaced or it's a dev's mistake? Is redirect required?

**Comment by dshevtsov** [comment]
@keharper Also,
```
src/pages/module-reference/module-quote-graph-ql.md
     54:3-54:133  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/remove-coupon.html is dead                                  no-dead-urls        remark-lint
     57:3-57:132  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/update-cart-items.html is dead                              no-dead-urls        remark-lint
     60:3-60:129  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/remove-item.html is dead                                    no-dead-urls        remark-lint
     63:3-63:146  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/set-shipping-address.html is dead                           no-dead-urls        remark-lint
     66:3-66:1[42](https://github.com/AdobeDocs/commerce-php/actions/runs/8602178073/job/23571128075#step:9:43)  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/set-billing-address.html is dead                            no-dead-urls        remark-lint
     69:3-69:1[43](https://github.com/AdobeDocs/commerce-php/actions/runs/8602178073/job/23571128075#step:9:44)  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/set-shipping-method.html is dead                            no-dead-urls        remark-lint
     72:3-72:140  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/set-payment-method.html is dead                             no-dead-urls        remark-lint
     75:3-75:134  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/set-guest-email.html is dead                                no-dead-urls        remark-lint
     78:3-78:152  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/set-payment-place-order.html is dead                        no-dead-urls        remark-lint
     81:3-81:121  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/merge-carts.html is dead                                    no-dead-urls        remark-lint
     84:3-84:121  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/place-order.html is dead                                    no-dead-urls        remark-lint
```

and

```
src/pages/module-reference/module-wishlist-graph-ql.md
      41:3-41:88  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/usage/wishlist.html is dead                                           no-dead-urls        remark-lint
     46:3-46:131  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/remove-products-from-wishlist.html is dead                  no-dead-urls        remark-lint
     47:3-47:127  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/mutations/update-products-in-wishlist.html is dead                    no-dead-urls        remark-lint
```

**Comment by dshevtsov** [comment]
@keharper I fixed most of the links in b08243c except those I couldn't find how to fix:

```
src/pages/module-reference/module-quote-graph-ql.md
    57:3-57:140  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-cart-items/ is dead              no-dead-urls  remark-lint
    81:3-81:129  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/merge-carts/ is dead                    no-dead-urls  remark-lint

src/pages/module-reference/module-wishlist-graph-ql.md
    46:3-46:139  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-products-from-wishlist/ is dead  no-dead-urls  remark-lint
    47:3-47:135  warning  Link to https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-products-in-wishlist/ is dead    no-dead-urls  remark-lint
```

**Comment by dshevtsov** [comment]
@jhadobe maybe you can help with the links☝️?

**Comment by keharper** [comment]
In `module-customer-graph-ql` the link on line 39 doesn't have a trailing slash `/`. When I copy and paste the link, it works. Maybe the link checker requires that slash

**Comment by keharper** [comment]
That README must have been writed when GraphQL was brand new. It doesn't make sense to only link to two queries. If you're updating README files, I suggest deleting the reference to `isEmailAvailable`. The `customer` query is not as concerning, but I'd rather link to https://developer.adobe.com/commerce/webapi/graphql/schema/customer/

File: src/pages/module-reference/module-wishlist-graph-ql.md [hunk 1]
```diff
@@ -24,24 +24,24 @@ Before installing this module, note that the Magento_WishlistGraphQl is dependen
...
-- [`wishlist`](https://devdocs.magento.com/guides/v2.4/graphql/queries/wishlist.html)
+- [`wishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/wishlist/queries/wishlist/)
 
 ### Mutations
 
-- [`addProductsToWishlist`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/add-products-to-wishlist.html)
-- [`removeProductsFromWishlist`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/remove-products-from-wishlist.html)
-- [`updateProductsInWishlist`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/update-products-in-wishlist.html)
+- [`addProductsToWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/wishlist/mutations/add-products/)
+- [`removeProductsFromWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-products-from-wishlist/)
```
Comment (keharper) [inline src/pages/module-reference/module-wishlist-graph-ql.md:46]:
```suggestion
- [`removeProductsFromWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-products/)
```

File: src/pages/module-reference/module-wishlist-graph-ql.md [hunk 2]
```diff
@@ -24,24 +24,24 @@ Before installing this module, note that the Magento_WishlistGraphQl is dependen
...
+- [`updateProductsInWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-products-in-wishlist/)
```
Comment (keharper) [inline src/pages/module-reference/module-wishlist-graph-ql.md:47]:
```suggestion
- [`updateProductsInWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-products/)
```

File: src/pages/module-reference/module-quote-graph-ql.md [hunk 3]
```diff
@@ -16,73 +16,72 @@ Before installing this module, note that the Magento_QuoteGraphQl is dependent o
...
-  [Learn more about setGuestEmailOnCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/set-guest-email.html).
+  [Learn more about setGuestEmailOnCart mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/set-guest-email/).
 
 - `setPaymentMethodAndPlaceOrder` mutation - sets the cart payment method and converts the cart into an order. **This mutation has been deprecated**. Use the `setPaymentMethodOnCart` and `placeOrder` mutations instead.
-  [Learn more about setPaymentMethodAndPlaceOrder mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/set-payment-place-order.html).
+  [Learn more about setPaymentMethodAndPlaceOrder mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/set-payment-place-order/).
 
 - `mergeCarts` mutation - transfers the contents of a guest cart into the cart of a logged-in customer.
-  [Learn more about mergeCarts mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/merge-carts.html).
+  [Learn more about mergeCarts mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/merge-carts/).
```
Comment (keharper) [inline src/pages/module-reference/module-quote-graph-ql.md:81]:
```suggestion
  [Learn more about mergeCarts mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/merge/).
```

File: src/pages/module-reference/module-quote-graph-ql.md [hunk 4]
```diff
@@ -16,73 +16,72 @@ Before installing this module, note that the Magento_QuoteGraphQl is dependent o
...
-  [Learn more about applyCouponToCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/apply-coupon.html).
+  [Learn more about applyCouponToCart mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/apply-coupon/).
 
 - `removeCouponFromCart` mutation - removes a previously-applied coupon from the cart.
-  [Learn more about removeCouponFromCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/remove-coupon.html).
+  [Learn more about removeCouponFromCart mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-coupon/).
 
 - `updateCartItems` mutation - allows you to modify items in the specified cart.
-  [Learn more about updateCartItems mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/update-cart-items.html).
+  [Learn more about updateCartItems mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-cart-items/).
```
Comment (keharper) [inline src/pages/module-reference/module-quote-graph-ql.md:57]:
```suggestion
  [Learn more about updateCartItems mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-items/).
```

File: src/pages/module-reference/module-wishlist-graph-ql.md (hunk 1 above)
Comment (dshevtsov) [reply src/pages/module-reference/module-wishlist-graph-ql.md:46]: https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-products/ is still 404

File: src/pages/module-reference/module-wishlist-graph-ql.md (hunk 2 above)
Comment (dshevtsov) [reply src/pages/module-reference/module-wishlist-graph-ql.md:47]: https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-products/ is still 404

File: src/pages/module-reference/module-wishlist-graph-ql.md (hunk 1 above)
Comment (keharper) [reply src/pages/module-reference/module-wishlist-graph-ql.md:46]: https://developer.adobe.com/commerce/webapi/graphql/schema/wishlist/mutations/remove-products/

File: src/pages/module-reference/module-wishlist-graph-ql.md (hunk 2 above)
Comment (keharper) [reply src/pages/module-reference/module-wishlist-graph-ql.md:47]: https://developer.adobe.com/commerce/webapi/graphql/schema/wishlist/mutations/update-products/

**Comment by dshevtsov** [comment]
@keharper, and the major question still remains. Do the fixed links require redirects?

**Pull request merged**